addons/CLib/MapGraphics/fn_mapGraphicsPosition.sqf

```diff
@@ -12,7 +12,7 @@
     1: Map <Control> (Default: controlNull)
 
     Returns:
-    Position <Position3D>
+    Position <Array> (Position3D)
```

Comment:
If you want to do it like this, it has to be:
`Position3D <Array>`

Reply:
Done